File: 2 May 24 shift/longest string/code.java
import java.util.Scanner;

public class Main {
    public static String getLongestString(String s) {
        String[] words = s.split("\\s+");
        String ans = "";
        int maxi = 0;
        for (String word : words) {
            if (word.length() > maxi) {
                ans = word;
                maxi = word.length();
            }
        }
        return ans;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String s = scanner.nextLine();
        System.out.println("The longest string is: " + getLongestString(s));
    }
}

File: 2 May 24 shift/longest string/code.py
def getLongestString(s):
    l = s.split()
    maxi = 0 
    ans = ""
    for word in l:
        if len(word) > maxi:
            ans = word 
            maxi = len(word)
    return ans 

s = input()
print("The longest string is:", getLongestString(s))